src/desktop/components/hover_pulldown/index.styl

```diff
@@ -76,7 +76,7 @@ computed-padding(font-size, target-line-height)
     .hover-pulldown-menu
     .hover-pulldown-static
       left inherit
-      right 10px
+      right 0px
```

Comment:
I tried to look where right aligned pull down is used and seems like the only 2 places is the nav bar person and bell icons. So updating this in the source instead of branching into more specific styles.

Reply:
👍 So the various pulldown menus are one of the key items to look closely in staging!